### DISCUSSION TITLE
Trouble with a sniper

### DISCUSSION THREAD

**Discussion by Expl0rer12** [marioparaschiv/nitro-sniper#405] [Q&A]
When I run the sniper it shows this text "Main token does not have a billing source, some codes will not be sniped.". How can I fix this?

**Comment by Craftzman7** [comment]
Add a payment method to your discord account to fix it. Note that if you've had nitro before you can ignore the warning.

**Reply by Expl0rer12** [reply]
Yay! It worked, thanks)
